src/components/input.tsx

```diff
@@ -0,0 +1,15 @@
+type InputProps = {
+  type: string;
+  checked: boolean;
+  style?: string;
+  onChange: (e: React.ChangeEvent<HTMLElement>) => void;
+};
+
```

Comment:
Since these props are directly tied to `<input>` we should try to match the prop name with the actual prop we're going to pass into.

In this case, `style` should be `className`